File: workspace/judoku/src/main/java/judoku/data/Line.java
package judoku.data;

public class Line extends AbstractCellSet {
	public Line(int id) {
		super(id);
	}

	protected String getName() {
		return "line";
	}
	
	public String toString() {
        StringBuffer buf = new StringBuffer();
    	for (Cell c: this.getCells()) {
    		buf.append(c);
    	}

        return buf.toString();
   }
}

File: workspace/scudoku/src/main/scala/scudoku/data/Line.scala
package scudoku.data

/**
 * Java2Scala port from judoku.data.Line
 */

//TODO rename to Row
class Line extends CellSet("line") {
  override def toString = {
      val buf: StringBuffer = new StringBuffer();
      for (c <- getCells)  buf.append(c)
      buf.toString
  }
}